biolink-model.yaml

```diff
@@ -4753,37 +4753,6 @@ classes:
   ## ATTRIBUTES
   ## ----------
 
-  ## Ontology Classes
```

Comment:
Why are we removing this section from the YAML?

Perhaps it would be safe to merge master branch to your `replace_commas` branch to pull in the latest changes.